quickstart.md

```diff
@@ -1,3 +1,23 @@
+# Table of Contents
```

Comment:
Nice table! I like it.

Reply:
This is standard practice in KEPs, and we should follow it too.
See https://github.com/kubernetes/enhancements/blob/master/hack/update-toc.sh for how KEPs do it.

Reply:
for now we can do this by hand as @nitinmewar has assembled here.  As we get more advanced/mature we can turn to automation.